astropy/cosmology/flrw.py

```diff
@@ -268,6 +268,13 @@ def __init__(self, H0, Om0, Ode0, Tcmb0=0.0*u.K, Neff=3.04, m_nu=0.0*u.eV,
         self._inv_efunc_scalar = self.inv_efunc
         self._inv_efunc_scalar_args = ()
 
+    @property
+    def _init_arguments(self):
+        ps = super()._init_arguments
+        if ps["m_nu"] is None:
```

Comment:
Ahh. This is from a followup PR. I didn't separate the two properly. Thanks for the catch. Yes, it will give different results for ``cosmo.to_format("astropy.model", method=...)``, but that's for a followup PR.

Reply:
Expect to see this exact diff soon :)

Reply:
But I'm removing it from here.